src/mon/MonMap.cc

```diff
@@ -319,6 +320,9 @@ void MonMap::decode(ceph::buffer::list::const_iterator& p)
     tiebreaker_mon = "";
     stretch_marked_down_mons.clear();
   }
+  if (struct_v >= 10) {
+    decode(quorum, p);
```

Comment:
Hmm, not sure we need to disambiguate between empty `quorum` and talking with too-old monitor (v9) that doesn't provide this information. In the latter case wrapping the set in `std::optional` should help.

Reply:
but anyway, v9 won't have the optional bit on, what will be the benefit of that?